code/modules/vending/wardrobes.dm

```diff
@@ -296,6 +338,11 @@
 					/obj/item/clothing/head/nun_hood = 1,
 					/obj/item/clothing/suit/holidaypriest = 1,
 					/obj/item/storage/fancy/candle_box = 2)
+	contraband = list(/obj/item/storage/crayons = 2,
+						/obj/item/toy/crayon/spraycan = 2,
+						/obj/item/storage/fancy/candle_box = 2)
+	premium = list(/obj/item/seeds/watermelon/holy = 1,
+					/obj/item/reagent_containers/food/drinks/bottle/holywater = 2)
```

Comment:
doesn't belong in a wardrobe but i guess where else do you put it for the chaplain